src/pinaclj/tasks/publish.clj

```diff
@@ -25,10 +25,15 @@
     (task/error (t :en :already-published (:path page)))
     (add-header src-root page time-fn)))
 
+(defn- read-validation [src-root page time-fn src-path]
```

Comment:
I wonder if there's a better way to structure this. If this continues we may end up with a whole load of nesting going on.
Either this may be a good place to introduce exception handlers, or we find a more elegant way to handle errors. Or maybe it's fine as it is. Do you have any thoughts on that?